package.json

```diff
@@ -31,11 +31,11 @@
   },
   "homepage": "https://github.com/fastify/fastify-diagnostics-channel#readme",
   "dependencies": {
-    "fastify-plugin": "^4.5.1"
+    "fastify-plugin": "^5.0.0-pre.fv5.1"
   },
   "devDependencies": {
     "@fastify/pre-commit": "^2.1.0",
-    "fastify": "^4.26.2",
+    "fastify": "5.0.0-alpha.3",
```

Comment:
pnpm and yarn seems to use the lowest matching version.
I get the same result in local computer.